### ISSUE TITLE
[ZE] Aplicar limite nos creditos do holiday

### ISSUE THREAD

**Issue by gabrielacms** [quasemago/zrageservers#192]
Aproveitando que ainda falta um tempo para o próximo holiday começar, eu queria sugerir de colocar um limite na quantidade de hcredits que o jogador consegue coletar ao todo durante os holidays. 

O hcredit é resetado em todo holiday, então não tem necessidade em pegar MUITO. Minha sugestão  é:
- Limite: colocar um cap na quantidade de hcredits que o jogador consegue farmar
- Quando o jogador atingir essa quantidade, vai continuar dropando gift para ele mas ele não consegue pegar mais. 
Creio que até pode equilibrar e deixar de fato fixo todos os preços

**Comment by hcjordao** [comment]
Acho uma boa. Mas limitar até que ponto. Conseguir comprar todos os itens ou conseguir comprar apenas uns deles.

**Comment by gabrielacms** [comment]
Eu estou pelo cel dai fica meio ruim de procurar algumas coisas no discord. Mas por exemplo, o Demo nesse último holiday pegou uns 5-6k de hcredit se eu nao me engano. E continuava pegando. O holiday para mim, não é algo para ser competitivo. Pegou uma certa quantidade e dai deixa os outros pegarem tb. 
Mas respondendo sua pergunta, por mim limita para conseguir comprar alguns. 
Ou colocamos um limite tb na quantidade de itens que vai ir pra store. Que dai com uns 2500 ou 3k o jogador já consegue comprar bastante coisas

**Comment by lhsatsuma** [comment]
> Acho uma boa. Mas limitar até que ponto. Conseguir comprar todos os itens ou conseguir comprar apenas uns deles.
O suficiente pra ele comprar metade do shop, nao tem necessidade da pessoa conseguir comprar tudo, tanto q nem vai usar...

**Comment by alminha021** [comment]
Cap = Soma de todos os cosmeticos base ( tags , auras, tracer e he) + preço da skin mais cara

infused e gabi tem muito trabalho fazendo as parada pra pouca gente comprar.

**Comment by alminha021** [comment]
Considerando que sao so 2 skins , cara quiser as 2 skins ele compra, mas dai nao vai ter como pegar o resto.

Tambem nao sou contra deixar pra pegar tudo tbm, cara teve o trabalhpo de ficar catando, pormim, whatever

**Comment by quasemago** [comment]
concordo com o @alminha021, o cap se for implementado deve ser suficiente para se comprar toda a loja e ao mesmo tempo resolve o problema do cara ficar farmando além do necessário e pegando os gifts de outros jogadores.

**Comment by danielhidalgojunior** [comment]
É um cap total? Tipo, não posso ganhar mais do que 5k ou é um cap pra previnir acumulação?

Se for um cap total, acho que seria bom ele não poder comprar tudo, assim damos mais valor nos itens que poderão ser Comprados... Alguns escolheram tal item, outros escolheram outro... Gera uma certa exclusividade. Poder comprar tudo vai fazer com que os itens sejam bem common, aposto que o povo nem vai acabar usando os Itens visto que o colega do lado comprou td q nem ele.
